src/util/log.d

```diff
@@ -298,49 +304,47 @@ abstract class Logger
     }
 }
 
-class FileLogger(alias Layout) : Logger
+class FileLogger(Layout) : Logger
 {
-    // must be static to be thread-local
-    private static Appender!(char[]) buffer;
-
     private File file;
 
     // store cause it's awkward to derive in fileDescriptor, which must be signal-safe
     private int fileno;
 
-    this(string name, uint levels = LogLevel.info.orAbove)
+    private Layout layout;
+
+    mixin CachedAppender;
```

Comment:
Whether a cached appender is to be used, should be a decision of the layout. Or a general default.
Why does the `FileLogger` gets a cached appender while the SyslogLogger does not?

Reply:
removed, but it was cause FileLogger's write calls were profiled and found to be expensive